new-evaluation.html

```diff
@@ -0,0 +1,563 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>New evaluation</title>
+		<link rel="stylesheet" type="text/css" href="new-evaluation.css">
+	</head>
+	<body style="background-color: #f7f7f7; margin: auto">
+		<header class="header">
+			<img id="logo" src="logo-v2.png">
+			<a style="vertical-align: middle; line-height: 70px" href="evaluations.html">Evaluations</a>
+			<a style="border-bottom: 9px solid #419fe2;vertical-align: middle; line-height: 70px; padding-bottom: 12px" href="new-evaluation.html">New Evaluation</a>
+			<a style="float: right; vertical-align: middle; line-height: 70px" href="login.html">Logout</a>
+		</header>
+			<div class="div_form">
+				<form method="post" action="">
+					<input class="input" type="text"  name="Candidate" placeholder="Candidate">
+					<input class="input" type="text" name="Interviewer" placeholder="Interviewer">
+					<input class="input" type="date" name="Date" value="04/22/2012">
+					<h2>Technical level</h2>
+					<table class="table">
+						<tr>
```

Comment:
Alignment is a bit off here. Mind adjusting for improved legibility, pretty please?